setup.py

```diff
@@ -116,7 +116,7 @@ def build_extensions(self):
         build_ext.build_extensions(self)
 
 
-if not os.environ.get("MOCK_DOCS", False):
+if not os.environ.get("SKIP_COMPILATION", False):
```

Comment:
Reminder - I believe this will need the environment variable to be updated in RTD.

Reply:
Also, maybe we should add docs testing in the CI tests :thinking: I expected the tests to fail because of this.

Reply:
I think I just added following [these](https://docs.readthedocs.io/en/stable/pull-requests.html) instructions.